docs/metrics.md

```diff
@@ -0,0 +1,175 @@
+		* Unlock their credentials
+		* Sync their credentials from the Firefox desktop browser
+* Does requiring a Firefox Account constitute a roadblock to adoption?
+	* What proportion of new Lockbox users are pre-existing Firefox Account users?
+	* What proportion of users start the Account sign-in process but never complete it?
+* Does adoption of Lockbox lead to adoption of Firefox Mobile browsers (e.g. Focus)?
+	* Do users set the default browser in Lockbox to be a Firefox browser?
+
+In addition to answering the above questions that directly concern actions in the app, we will also be analyzing telemetry emitted from the password manager that exists in the the Firefox desktop browser. These analyses will primarily examine whether users of Lockbox start active curation of their credentials in the desktop browser (Lockbox users will not be able to edit credentials directly from the app).
+
+## Collection
+
+Data will be collected using this library:
+
+https://github.com/mozilla-mobile/android-components/blob/master/components/service/telemetry/README.md
+
+We plan to submit two ping types, both of which are implemented by the component above.
+
+First is the [core ping](https://github.com/mozilla-mobile/android-components/blob/master/components/service/telemetry/src/main/java/org/mozilla/telemetry/ping/TelemetryCorePingBuilder.java), which contains information about the android version, architecture, etc of the device lockbox has been installed on:
+
+https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/core-ping.html
+
+The second is the [event ping](https://github.com/mozilla-mobile/android-components/blob/master/components/service/telemetry/src/main/java/org/mozilla/telemetry/ping/TelemetryEventPingBuilder.java) which allows us to record event telemetry:
+
+https://github.com/mozilla-mobile/focus-android/wiki/Event-Tracking-with-Mozilla%27s-Telemetry-Service
+
+TODO: link to the lockbox for android source code at the point where the pings and event values are defined (when that code exists).
+
+See [this](https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt) for the kotlin source code that Firefox Focus uses to define its telemetry events.
+
+Every event must contain `category`, `method` and `object` fields, and may optionally contain `value` and `extra` fields as well.
+
+Events related to specific credentials should have an opaque `item_id` in the extra field where possible.
+
+
+Finally, the `appName` metadata sent with each ping should always be 'Lockbox'.
+
+See here for more information on event schemas:
+
+https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#public-js-api
+
+## List of Proposed Events
+
+1. When the app starts up:
+	* `category`: action
+	* `method`: startup
+	* `object`: app
+	* `value`: null
+	* `extras`: null
+
+2. When locking/unlocking the app:
```

Comment:
do we want to distinguish between autolocking and manual locking?

Reply:
they aren't differentiated on iOS.